dir.targets

```diff
@@ -61,5 +61,6 @@
   <PropertyGroup Condition="$(AssemblyOriginatorKeyFile.Contains('Open.snk'))">
     <StrongNameSig />
     <DelaySign>false</DelaySign>
-  </PropertyGroup> 
+    <PublicSign Condition="'$(OS)' != 'Windows_NT'">true</PublicSign>
```

Comment:
I'm saying we still use the build task for doing our fake signing, see https://github.com/dotnet/buildtools/issues/1221, so we can probably get away with always setting PublicSign to false in all cases.